src/cdk/scrolling/scrolling.md

```diff
@@ -121,3 +121,10 @@ custom strategy by creating a class that implements the `VirtualScrollStrategy`
 providing it as the `VIRTUAL_SCROLL_STRATEGY` on the component containing your viewport.
 
 <!-- example(cdk-virtual-scroll-custom-strategy) -->
+
```

Comment:
It's been a while since I've looked at this code, but how is this different to setting the cache size to zero? I was under the impression that if a view can't be cached, it'll be recreated.

Reply:
I think the difference is that with cache size zero, it won't reuse already created views, but it *will* remove them from the DOM. The idea with this change is to have a mode where it never removes anything from the DOM, only adds.

Reply:
+1

Reply:
I updated the description to clarify this behavior.